src/sst/elements/vanadis/decoder/vdecoder.h

```diff
@@ -215,7 +215,6 @@ class VanadisDecoder : public SST::SubComponent
     void setThreadLocalStoragePointer(uint64_t new_tls) { tls_ptr = new_tls; }
 
     uint64_t getThreadLocalStoragePointer() const { return tls_ptr; }
-    uint64_t getCycleCount() const { return cycle_count; }
```

Comment:
Removing this would seem to break the MIPS32 implementation (see the change to the MIPS decoder). Is there a MIPS equivalent to Zicntr?

Reply:
Odd, I didn't see any use if this in the MIPS decoder. I wonder how I missed it. This change can be reverted, it was more of a bookkeeping "dead-code removal" choice and if that's wrong then obviously it can stick around. Or as you say I guess the MIPS register file could also have a cycle counter.

Reply:
I don't think that it's used anywhere right now but providing a way to get the cycle count for RV64 and not MIPS32 breaks the template.

This may take some internal discussion to resolve.

Reply:
I think if you want to retain this value, even though it's unused currently, a simple solution is to add the `cycle_count` member back into `VanadisMIPSDecoder` rather than the base class, and reinstall the `cycle_count = count` that I removed.